### PR TITLE
fix(ui): units are now kg and kWh

### PR DESCRIPTION
**Pull request by inimaz**
- Leave all graphs with units in kg and kWh.
- Emission comparisons are using now the ones from https://github.com/mlco2/codecarbon/pull/564.

### CODE REVIEW

**Comment by inimaz** [comment]
Note that now the graphs and comparisons are very small

**Comment by SaboniAmine** [comment]
Thanks Inigo! Indeed the artificial x1000 factor was added to visualize graphs for dev.

**Review by SaboniAmine — APPROVED** [review]

**Review by benoit-cty — APPROVED** [review]
Thanks !
Here is what my dashboard looks like, now :
image

**Review by benoit-cty — CHANGES_REQUESTED** [review]
On the page we say "days Of watching TV" but here we compute minutes of TV :
`const TV_CO2_EMISSIONS_PER_MINUTE = ((100 / 1000) * 0.475) / 60;`

**Comment by inimaz** [comment]
> On the page we say "days Of watching TV" but here we compute minutes of TV : `const TV_CO2_EMISSIONS_PER_MINUTE = ((100 / 1000) * 0.475) / 60;`

Nice catch thanks @benoit-cty ! I updated it here --> https://github.com/mlco2/codecarbon/pull/728/commits/13fc80c90566e2a0d4339e5530c67153fbe8d99b

**Review by benoit-cty — APPROVED** [review]
Very nice !

File: webapp/src/app/(dashboard)/[organizationId]/projects/[projectId]/page.tsx [hunk 1]
```diff
@@ -246,11 +220,10 @@ export default function ProjectPage({
                             </div>
                             <div className="flex flex-col col-span-2 justify-center">
                                 <p className="text-4xl text-primary">
-                                    {convertedValues.household} %
+                                    {convertedValues.citizen} %
                                 </p>
                                 <p className="text-sm font-medium">
-                                    Of an american household weekly energy
-                                    consumption
+                                    Of a U.S citizen weekly energy emissions
```
[the remark this answers is not in the document]
Comment (inimaz) [reply webapp/src/app/(dashboard)/[organizationId]/projects/[projectId]/page.tsx:226]: Didn't see this one!

[the remark this answers is not in the document]
Comment (SaboniAmine) [reply webapp/src/app/(dashboard)/[organizationId]/projects/[projectId]/page.tsx:226]: i'll fix it in #729

**Pull request merged**